editor/src/core/layout/layout-helpers.ts

```diff
@@ -278,11 +278,19 @@ export const LayoutHelpers = {
   setLayoutAttribute: (newValue: LayoutSystem) => (
     props: JSXAttributes,
   ): Either<string, JSXAttributes> => {
-    return setJSXValueAtPath(
-      props,
-      createLayoutPropertyPath('LayoutSystem'),
-      jsxAttributeValue(newValue),
-    )
+    if (newValue === LayoutSystem.PinSystem) {
```

Comment:
hah sneaky

Reply:
Looks like the only other value this can be called with is `LayoutSystem.Group`, which we don't support anyway, so please can you remove this and update the code that calls it to reflect that.

Reply:
fixed!